### PR TITLE
Now Incident templates will be effective when add or edit a incident

### PR DESCRIPTION
**Pull request by phecho**

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Please squash to one commit.

**Comment by phecho** [comment]
@GrahamCampbell Is this OK?

File: app/Http/Controllers/Dashboard/IncidentController.php [hunk 1]
```diff
@@ -111,14 +111,14 @@ public function createIncidentAction()
...
-                Binput::get('name'),
-                Binput::get('status'),
-                Binput::get('message'),
-                Binput::get('visible', true),
-                Binput::get('component_id'),
+                Binput::get('incident.name'),
+                Binput::get('incident.status', 0),
+                Binput::get('incident.message'),
+                Binput::get('incident.visible', true),
+                Binput::get('component_id', 0),
```
Comment (jbrooksuk) [inline app/Http/Controllers/Dashboard/IncidentController.php]: I've got a feeling that this should be `null`.

File: app/Http/Controllers/Dashboard/IncidentController.php [hunk 2]
```diff
@@ -111,14 +111,14 @@ public function createIncidentAction()
     {
         try {
             $incident = $this->dispatch(new ReportIncidentCommand(
-                Binput::get('name'),
-                Binput::get('status'),
-                Binput::get('message'),
-                Binput::get('visible', true),
-                Binput::get('component_id'),
+                Binput::get('incident.name'),
+                Binput::get('incident.status', 0),
```
Comment (jbrooksuk) [inline app/Http/Controllers/Dashboard/IncidentController.php]: Why are we defaulting the status? That's a value that should *always* be passed through?

File: app/Http/Controllers/Dashboard/IncidentController.php (hunk 1 above)
Comment (phecho) [inline app/Http/Controllers/Dashboard/IncidentController.php]: I have modified the corresponding template files.

File: .travis.yml [hunk 3]
```diff
@@ -1,11 +1,11 @@
 language: php
 
 php:
+  - 7.0
```
Comment (jbrooksuk) [inline .travis.yml]: What is this change about?

**Comment by jbrooksuk** [comment]
I'm not sure what's happened here... There seems to be a lot of changes exceeding what the title says it does?

**Comment by jbrooksuk** [comment]
Ideally these would be split into several PR's.

File: .travis.yml (hunk 3 above)
Comment (GrahamCampbell) [inline .travis.yml]: The travis changes need reverting, yeh.

File: app/Handlers/Commands/ComponentGroup/AddComponentGroupCommandHandler.php [hunk 4]
```diff
@@ -27,8 +27,9 @@ class AddComponentGroupCommandHandler
     public function handle(AddComponentGroupCommand $command)
     {
         $group = ComponentGroup::create([
-            'name'  => $command->name,
-            'order' => $command->order,
+            'name'         => $command->name,
+            'description'  => $command->description,
```
Comment (GrahamCampbell) [inline app/Handlers/Commands/ComponentGroup/AddComponentGroupCommandHandler.php]: too many spaces

File: app/Models/ComponentGroup.php [hunk 5]
```diff
@@ -42,8 +53,8 @@ class ComponentGroup extends Model
      * @var string[]
      */
     public $rules = [
-        'name'  => 'required|string',
-        'order' => 'int',
+        'name'        => 'required|string',
```
Comment (GrahamCampbell) [inline app/Models/ComponentGroup.php]: no

File: app/Models/ComponentGroup.php [hunk 6]
```diff
@@ -42,8 +53,8 @@ class ComponentGroup extends Model
...
+        'order'       => 'int',
```
Comment (GrahamCampbell) [inline app/Models/ComponentGroup.php]: no

**Comment by GrahamCampbell** [comment]
> 100644 → 100755

Please revert those changes/

File: .travis.yml (hunk 3 above)
Comment (phecho) [inline .travis.yml]: PHP 7 is up to twice as fast as PHP 5.x 
We can got the tips first time if the unit test is't passed.

Comment (GrahamCampbell) [inline .travis.yml]: That change is still unrelated so can't go in the same PR. Please ensure PRs are atomic.

**Comment by phecho** [comment]
How about my changes?
sorry for my poor experience to use github.

**Comment by jbrooksuk** [comment]
Can you split them into different Pull Requests please? This PR makes a lot of changes that would be hard to revert individually.

**Comment by phecho** [comment]
I don't know how split them, it seems a very difficult task. 
sad.

**Comment by GrahamCampbell** [comment]
You can cherry-pick commits, or just manually make changes on other branches.

**Pull request closed without merging**